OpenRA.Platforms.Default/Sdl2HardwareCursor.cs

```diff
@@ -45,9 +45,6 @@ public Sdl2HardwareCursor(Size size, byte[] data, int2 hotspot)
 				// This call very occasionally fails on Windows, but often works when retried.
 				for (var retries = 0; retries < 3 && Cursor == IntPtr.Zero; retries++)
 					Cursor = SDL.SDL_CreateColorCursor(surface, hotspot.X, hotspot.Y);
-
-				if (Cursor == IntPtr.Zero)
-					throw new Sdl2HardwareCursorException($"Failed to create cursor: {SDL.SDL_GetError()}");
```

Comment:
We can now remove the `Sdl2HardwareCursorException` class.

Reply:
Thanks I forgot that, removed now.